src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -64,8 +65,8 @@ public JsonAdaptedPerson(Person source) {
                             .map(JsonAdaptedTag::new)
                             .collect(Collectors.toList()));
         grades.addAll(source.getGradeList().getList().stream() // Convert GradeList to JSON
-                .map(JsonAdaptedGrade::new)
-                .collect(Collectors.toList()));
+                              .map(JsonAdaptedGrade::new)
+                              .collect(Collectors.toList()));
```

Comment:
Why 16 spaces?

Reply:
Because it is a continuation of the source.*, so it should align?